tools/racon/macros.xml

```diff
@@ -0,0 +1,23 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+        <requirement type="package" version="1.3.1">racon</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.1101/gr.214270.116</citation>
+            <citation type="bibtex">
+@misc{githubracon,
```

Comment:
the bibtex can be remove if you have a DOI

Reply:
bibtex is for the github link

Reply:
Yes, but we assume this is mentioned in the paper.